README.md

```diff
@@ -168,6 +168,17 @@ $ vault get list_secret
 - secret3
 ```
 
+### Write a secret using an invisible input prompt
```

Comment:
Would not that be better to add this section before or after `Write a secret via stdin` section ?

Reply:
done